### PR TITLE
Adds move_base_topic option

### PR DESCRIPTION
**Pull request by BarceloChristian**
As `create_autonomy` uses move_base under a namespace, and this package had a fixed value for the topic, this PR adds the option to set the topic to call MoveBase.

### CODE REVIEW

File: explore/launch/explore.launch [hunk 1]
```diff
@@ -1,15 +1,16 @@
 <launch>
 <node pkg="explore_lite" type="explore" respawn="false" name="explore" output="screen">
-  <param name="robot_base_frame" value="base_link"/>
+  <param name="robot_base_frame" value="create1_tf/base_footprint"/>
```
Comment (eborghi10) [inline explore/launch/explore.launch]: Can you move all these params to a YAML file?

Comment (eborghi10) [inline explore/launch/explore.launch]: BTW, this is a robot-agnostic package so you shouldn't make this change here.

In fact, you should make these changes in the `create_autonomy` repo. And the one that I mentioned about the YAML too.

File: explore/launch/explore.launch [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
   <param name="costmap_topic" value="map"/>
   <param name="costmap_updates_topic" value="map_updates"/>
   <param name="visualize" value="true"/>
   <param name="planner_frequency" value="0.33"/>
   <param name="progress_timeout" value="30.0"/>
   <param name="potential_scale" value="3.0"/>
   <param name="orientation_scale" value="0.0"/>
-  <param name="gain_scale" value="1.0"/>
-  <param name="transform_tolerance" value="0.3"/>
+  <param name="gain_scale" value="0."/>
```
Comment (eborghi10) [inline explore/launch/explore.launch]: What are these changes about?

**Review by eborghi10 — CHANGES_REQUESTED** [review]

**Comment by eborghi10** [comment]
> _No description provided._

Can you provide a short description?

File: explore/launch/explore.launch (hunk 1 above)
Comment (BarceloChristian) [reply explore/launch/explore.launch]: I removed the mentions to the robot in the code, and then the needed changes will be applied from a YAML in create_autonomy.

File: explore/launch/explore.launch (hunk 2 above)
Comment (BarceloChristian) [reply explore/launch/explore.launch]: I removed these changes and will call them from create_autonomy, but summarizing:
`potential_scale` gives weigh to the nearer frontiers while
`gain_scale` gives weigh to the bigger frontiers.
Doing `gain_scale = 0` the robot will try to continue exploring always against the closest frontier, I realized that it's the best way.

**Comment by BarceloChristian** [comment]
@eborghi10 Can you give it another try?

File: explore/launch/explore.launch (hunk 2 above)
Comment (eborghi10) [reply explore/launch/explore.launch]: Put this explanation in `create_autonomy` PR, please.

**Review by eborghi10 — APPROVED** [review]

**Pull request merged**